java/code/src/com/redhat/rhn/manager/kickstart/cobbler/CobblerXMLRPCHelper.java

```diff
@@ -69,7 +69,7 @@ public Object invokeMethod(String procedureName, List args) throws XmlRpcFault {
             retval = client.invoke(procedureName, args);
         }
         catch (XmlRpcException e) {
-            throw new RuntimeException("XmlRpcException calling cobbler.", e);
+            throw new RuntimeException("XmlRpcException calling cobbler. Cobbler might not be running.", e);
```

Comment:
In general we should avoid using `RuntimeException` this the compiler doesn't enforce handling them...